pytest_plugins/rerun_rp/rerun_user.py

```diff
@@ -0,0 +1,66 @@
+import logging
+
+from robottelo.config import settings
+from robottelo.report_portal.portal import ReportPortal
+
+LOGGER = logging.getLogger(__name__)
+
+
+def _get_user_tests(launch, user):
```

Comment:
This function is almost indistinguishable from `_get_skipped_tests` in other file - the only difference is in parameters passed to `launch.tests(user=user)`.

Consider single function that passes extra arguments to `launch.tests`:
```python
def _get_tests(launch, **kwargs):
    tests = launch.tests(**kwargs).keys()
    tests = [str(test).replace('::', '.') for test in tests]
    return tests
```

or additional function responsible for transformation of test names:

```python
def _transform_rp_names_to_pytest(tests):
    return [str(test).replace('::', '.') for test in tests]

def _get_user_tests(launch, user):
    tests = launch.tests(user=user).keys()
    return _transform_rp_names_to_pytest(tests)
```